common/source/docs/common-frsky-flightdeck.rst

```diff
@@ -0,0 +1,43 @@
+.. _common-frsky-flightdeck:
```

Comment:
Perhaps I wasn't clear on my comment. All the equipment pages describe how to connect peripherals to ArduPilot, as send the users to the specific vendor page for more info. Some of the pages don't even refer to any specific vendor like the case of radio transmitters http://ardupilot.org/copter/docs/common-rc-transmitter-flight-mode-configuration.html
The Flightdeck option as such is a product developed to use one of the telemetry options available on ArduPilot, so not directly necessary to use the telemetry and a brief description and link to the vendor should be enough on the end of the page where the specific telemetry option is described/documented. IMHO.